source/common/grpc/transcoder_input_stream_impl.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "common/buffer/buffer_impl.h"
+
+#include "src/transcoder_input_stream.h"
```

Comment:
https://github.com/grpc-ecosystem/grpc-httpjson-transcoding/issues/9 for prefix.

Probably not need mocks, just like buffers, it is hard to `EXPECT` how the input stream is used, so just using real impl should be fine.